CHANGELOG

```diff
@@ -1,3 +1,7 @@
+#1.4.1
```

Comment:
We generally try to follow [semver](http://semver.org/) so the last digit is generally for bug fixes, since this is new functionality it should be `1.5.0` (unless it is a breaking change)